### PR TITLE
[registration] Correct documentation in ia_fpcs

### PR DESCRIPTION
**Pull request by IpandaZll**
Problem is mentioned here -- [#4534](https://github.com/PointCloudLibrary/pcl/issues/4534)

### CODE REVIEW

File: registration/include/pcl/registration/ia_fpcs.h [hunk 1]
```diff
@@ -485,7 +485,7 @@ class FPCSInitialAlignment : public Registration<PointSource, PointTarget, Scala
   float delta_;
 
   /** \brief Score threshold to stop calculation with success.
-   * If not set by the user it is equal to the approximated overlap
+   * If not set by the user it dpends on the size of the approximated overlap
```
Comment (JStech) [inline registration/include/pcl/registration/ia_fpcs.h:488]:
```suggestion
   * If not set by the user it depends on the size of the approximated overlap
```

Comment (IpandaZll) [reply registration/include/pcl/registration/ia_fpcs.h:488]: Thank you so much! It was my fault.

**Review by mvieth — APPROVED** [review]
Thanks @IpandaZll !

**Comment by IpandaZll** [comment]
> Thanks @IpandaZll !

I am happy to contribute to this project.

**Review by larshg — APPROVED** [review]

**Pull request merged**